added cookies policy link to footer and the cookies policy modal

luisramos0 opened the pull request:
#### What? Why?


#### What should we test?


#### Release notes

myriamboure commented:
Wahou, that's a fast first bit! I like that :-) Great reactivity @luisramos0 !

myriamboure commented:
Ok normally we say one PR for one issue, but maybe I should have opened only one issue for the two things...

luisramos0 commented:
I have decided to close this PR and move these 2 commits to PR #2411 
@sauloperez these 2 commits here are also there:
- Changed modals max height from 120% to 100%.
and
- Added new cookies policy page and added a link to it in the footer 
The other PR 2411 has 4 more commits:
https://github.com/openfoodfoundation/openfoodnetwork/pull/2411/commits
